pkgs/top-level/python-packages.nix

```diff
@@ -4684,6 +4684,12 @@ in {
 
   Pyro = callPackage ../development/python-modules/pyro { };
```

Comment:
How about removing the old version then?

Reply:
It seems that openopc uses that :-/ https://github.com/NixOS/nixpkgs/blob/92a047a6c4d46a222e9c323ea85882d0a7a13af8/pkgs/tools/misc/openopc/default.nix#L4

Reply:
This project was updated last in 2015. I will propose the removal from nixpkgs.

Reply:
https://github.com/NixOS/nixpkgs/pull/49362

Reply:
The package was removed.